Apps/phradar/radar.json

```diff
@@ -0,0 +1,550 @@
+{
+    "appid": "f8fbcb9f-fc58-438c-9e39-14f034279b29",
+    "name": "Radar",
+    "description": "This app integrates with the Radar Incident Response Platform",
+    "type": "incidenting",
```

Comment:
Yeah it's not the most apparent, we had considered `ticketing` as well, but didn't feel that was accurate. I am having conversations with our PM about this type as well, your suggestions would be appreciated though.

Reply:
Got a response from PM, would using `privacy incident response` work for our app type?

Reply:
I've looked through our existing categories and its hard to say if there is a fit. Because we don't have published guidelines, 'privacy incident response' is ok with me, but I think 'incident response' or even just 'response' would be better for searchability.

Reply:
Right, I can understand the need for better keyword searching. Our product team would still like to keep the keyword `privacy` in the app type since it's the core purpose of Radar. If it's fine to update the app type to `privacy incident response` for now we think we will give that a try, and continue the conversation if this becomes an issue.